bin/ci

```diff
@@ -94,14 +94,25 @@ with_build_env() {
   export TZ="America/New_York"
 
   on_linux verify_linux_environment
+
+  export DOCKER_TEST_PREFIX="crystallang/crystal:0.24.2"
```

Comment:
Instead of hardcoding the previous stable version here, could we use a special tag, like `latest_stable` for that?

Reply:
... but then later if we want to build an old commit, it won't work as it will use the latest stable at current time, not at time of commit.

Maybe not a good idea in the end if we want to support old commit build. Right?